content/oci/storage.go

```diff
@@ -106,6 +106,19 @@ func (s *Storage) Push(_ context.Context, expected ocispec.Descriptor, content i
 	return nil
 }
 
+// Delete removes the target blob.
+func (s *Storage) Delete(ctx context.Context, target ocispec.Descriptor) error {
+	path, err := blobPath(target.Digest)
+	if err != nil {
+		return fmt.Errorf("%s: %s: %w", target.Digest, target.MediaType, errdef.ErrInvalidDigest)
+	}
+	targetPath := filepath.Join(s.root, path)
+	if _, err := os.Stat(targetPath); errors.Is(err, os.ErrNotExist) {
```

Comment:
This Stat call doesn't appear to be necessary. Can't we just check the result of the os.Remove call instead?